ext/stackprof/stackprof.c

```diff
@@ -130,8 +146,9 @@ stackprof_start(int argc, VALUE *argv, VALUE self)
     _stackprof.interval = interval;
     _stackprof.out = out;
 
-    if (raw) {
-	gettimeofday(&_stackprof.last_sample_at, NULL);
+    if (_stackprof.debug) {
+        printf("started with interval %d (%ld sec %d usec)\n",
```

Comment:
(This highlights the separate issue of https://github.com/tmm1/stackprof/issues/125)